mobile-menu.js

```diff
@@ -0,0 +1,45 @@
+const mobileMenu = `
+<div id="close">
+  <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path fill-rule="evenodd" clip-rule="evenodd"
+      d="M0.292893 0.292893C0.683417 -0.0976311 1.31658 -0.0976311 1.70711 0.292893L7 5.58579L12.2929 0.292893C12.6834 -0.0976311 13.3166 -0.0976311 13.7071 0.292893C14.0976 0.683417 14.0976 1.31658 13.7071 1.70711L8.41421 7L13.7071 12.2929C14.0976 12.6834 14.0976 13.3166 13.7071 13.7071C13.3166 14.0976 12.6834 14.0976 12.2929 13.7071L7 8.41421L1.70711 13.7071C1.31658 14.0976 0.683417 14.0976 0.292893 13.7071C-0.0976311 13.3166 -0.0976311 12.6834 0.292893 12.2929L5.58579 7L0.292893 1.70711C-0.0976311 1.31658 -0.0976311 0.683417 0.292893 0.292893Z"
+      fill="#93A79D" />
+  </svg>
+</div>
+<nav>
+  <ul>
+    <li>
+      <a href="#recent-works-start">Portfolio</a>
+    </li>
+    <li>
+      <a href="#about-start">About</a>
+    </li>
+    <li>
+      <a href="#contact-start">Contact</a>
+    </li>
+  </ul>
```

Comment:
Two of the project requirements state that:
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

Currently, none of these requirements are met in this pull request. I would like to suggest that you crosscheck the `href` attributes of the `a` tags and be sure they link to the correct sections in the project. 

I also observed that your links do not have `eventlistener` added to them. It would be nice if you added an `eventlistener` to them and `onClick`, making the mobile menu disappear.